### DISCUSSION TITLE
Unable to login inside the editor

### DISCUSSION THREAD

**Discussion by sriranggd** [zed-industries/zed#6476] [Feature Requests]
Hi,

I am able to download and run the Zed editor on my machine, however the "Sign in" inside the editor is not working.

Is it because the primary email address in my github account is different from the email address to which I received the Zed invite?

My personal (gmail) id is the primary address and my work email id is an additional address in my github account. I have received the invite to my work email address. Any way for me to login to the editor in such a scenario?

**Comment by JosephTLyons** [comment]
Hey @sriranggd, sorry for the long wait here.  Not entirely sure what might be going on.  If you could [email us](hi@zed.dev) with:

- The email address you received the invite with
- The email address you want to be associated with 
- Your GitHub handle

We could take a look around and try to see what might be happening.

**Comment by JosephTLyons** [comment]
A combination of us correcting some data in the database and @sriranggd's clearing the cookies in their browser fixed the issue.
